src/blockchain/src/block.rs

```diff
@@ -78,19 +78,19 @@ impl Block {
 
 #[derive(Serialize, Deserialize, Debug, Clone, Hash, PartialEq, Eq)]
 pub struct Transaction {
-    /// - `trans_type` Pretty obvious. Used to help decode the rest of the transaction
+    /// Pretty obvious. Used to help decode the rest of the transaction
     pub trans_type: TransactionType,
-    /// - `submitter` The unique identifier of the participant on the network
+    /// The unique identifier of the participant on the network
     pub submitter: Address,
-    /// - `timestamp` When the transaction was submitted in ms from the epoch
+    ///  When the transaction was submitted in ms from the epoch
     pub timestamp: u64,
-    /// - `payload` User payload - no decoding will be supplied
+    /// User payload no decoding will be supplied
     pub payload: Vec<u8>,
-    /// - `nonce` A random value added to the transaction payload to harden it
+    /// A random value added to the transaction payload to harden it
     nonce: u128, // Adds a salt to harden
-    /// - `transaction_hash` A hash of all the fields in the transaction
+    /// A hash of all the fields in the transaction
     pub transaction_hash: HashDigest,
-    /// - `signature` The signature of the transaction_hash data in this transaction
+    /// The signature of the transaction_hash data in this transaction
```

Comment:
I was reading online there a way to link to other fields, could you mind playing with that?

Reply:
Let me see